### PR TITLE
[gazebo_ros] Remove ROS remapping arguments from gazebo_ros launch scripts.

### PR DESCRIPTION
**Pull request by peci1**
Solves issue https://github.com/ros-simulation/gazebo_ros_pkgs/issues/387 .

### CODE REVIEW

**Comment by j-rivero** [comment]
Could you please merge the current jade-devel branch so we get some testing from the travis service? Thanks.

**Comment by peci1** [comment]
Here we go...

File: gazebo_ros/scripts/debug [hunk 1]
```diff
@@ -1,6 +1,10 @@
 #!/bin/sh
 final="$@"
 
+# remove ros remapping arguments like "__name:=node_name"; otherwise, they'd 
+# be interpreted as a world name by gzserver, and it'd crash while loading it
+final=`echo "$final" | sed 's/[^ ]*:=[^ ]* \?//g'`
```
Comment (j-rivero) [inline gazebo_ros/scripts/debug:6]: we probably don't need the echo call, does `sed 's/[^ ]*:=[^ ]* \?//g' <<< $final` works for you?

Comment (peci1) [inline gazebo_ros/scripts/debug:6]: Yes, it works. But I haven't known something like `<<<` exists. Isn't it more readable to do it with the `echo` call?

Comment (j-rivero) [inline gazebo_ros/scripts/debug:6]: > Isn't it more readable to do it with the echo call?

I would say that is a subjective matter but we are saving one extra call and only run grep. minimal minimal peformance impact but better use of tools, IMHO. I leave it up to you.

Comment (peci1) [inline gazebo_ros/scripts/debug:6]: Okay. Then you know my preference :)

**Comment by j-rivero** [comment]
Thanks for the patch!

**Pull request merged**